tests/src/system/basic/WskBasicTests.scala

```diff
@@ -333,6 +333,26 @@ class WskBasicTests
             }
     }
 
+    /**
+      * Tests creating an nodejs action that throws a whisk.error() response. The error message thrown by the
+      * whisk.error() should be returned.
+      */
+    it should "create and invoke a blocking action resulting in a whisk.error response" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "whiskError"
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some(TestUtils.getTestActionFilename("applicationError1.js")))
+            }
+
+            if (usePythonCLI) {
+                wsk.action.invoke(name, blocking = true, expectedExitCode = 246)
+                  .stderr should include regex (""""error": "This error thrown on purpose by the action."""")
+            } else {
+                wsk.action.invoke(name, blocking = true)
```

Comment:
Why not the same exit code? 0 exit code implies no error. The python cli returns the http rest status as the exit code on failures. What's the rationale for doing this differently?